gpt4all-bindings/typescript/README.md

```diff
@@ -128,6 +129,7 @@ yarn test
 *   why your model may be spewing bull 💩
     *   The downloaded model is broken (just reinstall or download from official site)
     *   That's it so far
+* If you gpu fails to init, ensure [vulkan sdk](https://vulkan.lunarg.com/#new_tab) is installed.
```

Comment:
This makes it sound like a runtime dependency, but it's actually only a build-time dependency - and Kompute defaults to downloading its own Vulkan headers anyway so I'm not sure that's even the case.

Reply:
Ah, in that case I'll remove this note.